src/main/scala/org/hatdex/hat/api/json/DataStatsFormat.scala

```diff
@@ -43,8 +43,8 @@ trait DataStatsFormat extends DataDebitFormats {
   implicit protected val dataStorageStatsFormat: Format[DataStorageStats]     = Json.format[DataStorageStats]
   implicit protected val inboundDataStatsFormat: Format[InboundDataStats]     = Json.format[InboundDataStats]
   implicit protected val outboundDataStatsFormat: Format[OutboundDataStats]   = Json.format[OutboundDataStats]
-  implicit protected val richDataDebitFormat                                  = RichDataJsonFormats.richDataDebitFormat
-  implicit protected val dataDebitFormat                                      = RichDataJsonFormats.dataDebitFormat
+  implicit protected val richDataDebitFormat: Format[RichDataDebit]                                  = RichDataJsonFormats.richDataDebitFormat
```

Comment:
Is it just me who thinks that this looks wrong? The "=" character is floating in the middle of the line

Reply:
See below.